### DISCUSSION TITLE
Do not fill terminal with ~

### DISCUSSION THREAD

**Discussion by pabgan** [sharkdp/bat#2928] [Q&A]
Hello. I am sure this is explained somewhere but I could not find it, probably I am not using the right keywords

The thing is: 

- If I use `cat`:

```
pabgan@roBin ~/dzs/releases/demo/24.2/tickets/DSLE-26509/t1
 (master *) % ls
backupator.sql  config  out  setup.cfg  test.md
pabgan@roBin ~/dzs/releases/demo/24.2/tickets/DSLE-26509/t1
 (master *) % cat setup.cfg 
modules_affected=(pe)
config_dir=config
sh_script=config.sh
db_script=config.sql
```

- If I use `bat`:

```
pabgan@roBin ~/dzs/releases/demo/24.2/tickets/DSLE-26509/t1
 (master *) % ls
backupator.sql  config  out  setup.cfg  test.md
pabgan@roBin ~/dzs/releases/demo/24.2/tickets/DSLE-26509/t1
 (master *) % cat setup.cfg 
modules_affected=(pe)
config_dir=config
sh_script=config.sh
db_script=config.sql
~
~
~
~
~
~
~
~
~
```

And that `~` thing goes on until I can't see the previous command.

I want to turn that behavior off and use `bat` just as a colorized `cat`, but I cannot find the option or instructions.

Thank you very much.

**Comment by keith-hall** [comment]
Presumably this comes from your pager, so adding `--paging=never` to your `bat` config file or command line should prevent this from happening.

**Comment by pabgan** [comment]
Thanks, @keith-hall . With that hint I have figured out what was happening:

1. `bat`: prints decorations AND fills the screen with `~`
1. `bat --paging=never`: prints decorations WITHOUT filling the screen with `~`
1. `bat --paging=never --plain`: NO decorations AND fills the screen with `~`   **<==== this is what I was using**
1. `bat --paging=never --style=plain`: NO decorations WITHOUT filling the screen with `~` **<==== this is what I wanted**
1. `bat --plain --plain`: NO decorations WITHOUT filling the screen with `~`

The latest is what I wanted. Not sure if option 3 is different from 4 and 5 on purpose...
